src/memgraph.cpp

```diff
@@ -534,8 +535,9 @@ int main(int argc, char **argv) {
     if (!(is_valid_coordinator_instance || is_valid_data_instance)) {
       throw std::runtime_error(
           "You specified invalid combination of HA flags to start coordinator instance or data instance."
-          "Coordinator must be started with coordinator_id, port and management_port. Data instance must be "
-          "started with only management port.");
+          "Coordinator must be started with coordinator_id, coordinator_hostname, coordinator_port and "
+          "management_port. Data instance must be "
+          "started only with management port.");
```

Comment:
looks like you branched from a branch and not main. just rebase

Reply:
yes intentionally, waiting for branch to be merged then I rebase on master